swan/values.yaml

```diff
@@ -1,8 +1,56 @@
+#
+# CVMFS access
+# - deployDaemonSet deploys CVMFS pods exposing `/cvmfs` path on the host.
+#     Access to CVMFS is provided by bind-mounting `/cvmfs` from the host.
+# - deployCsiDriver deploys a cluster-wide storage driver for CVMFS.
+#     Access to CVMFS is provided by persistent volume claims.
+# - useCsiDriver has to be used in case the hosting infrastructure provides
+#       a CSI driver to access CVMFS (i.e., it is not needed to deploy additional pods).
+#     Access to CVMFS is provided by persistent volume claims (identical to deployCsiDriver).
+# - repositories defines which CVMFS repos have to be mounted into singleusers' pods.
+#       The value passed depends on the method used, DaemonSet VS CSI driver.
+#
+# Defaults best support a stand-alone small deployment:
+# - Deploy a DaemonSet pod with the CVMFS client running inside
+# - Chunks from upstream are fetched connecting to the CERN Stratum 1 server
+#
+# Warning:
+# - It is discouraged to enable more than one at once.
+# - By setting all to false, access to EOS will not be possible
+#     and singleuser's session will not be able to start.
+#
 cvmfs:
+  deployDaemonSet: &cvmfsDeployDS true
```

Comment:
Can you also update `swan-cern` and the dev values?

Reply:
Fixed by 36ab2639dd48fca47ad680239a7c0332c4ec16d1